logstash-input-tcp.gemspec

```diff
@@ -1,13 +1,14 @@
 Gem::Specification.new do |s|
   s.name          = 'logstash-input-tcp'
-  s.version         = '4.1.2'
+  s.version       = ::File.read('version').split("\n").first
```

Comment:
I've seen this pattern used in beats, dissect and others as well. https://github.com/logstash-plugins/logstash-input-beats/blob/master/logstash-input-beats.gemspec#L1

Just a side note that we'll need to consolidate these approaches at some point...